BunqSdk/Exception/ForbiddenException.cs

```diff
@@ -2,7 +2,8 @@
 {
     public class ForbiddenException : ApiException
```

Comment:
Docs *Same for other exception types*

Reply:
This code has not been modified in this pr. Please create a follow up issue for this. This would need to be refactored in the other SDK's as well.